src/tdlib/chat.rs

```diff
@@ -236,11 +247,12 @@ impl Chat {
             ChatUnreadMentionCount(update) => {
                 self.set_unread_mention_count(update.unread_mention_count)
             }
-            DeleteMessages(_)
-            | MessageContent(_)
-            | MessageEdited(_)
-            | MessageSendSucceeded(_)
-            | NewMessage(_) => {
+            DeleteMessages(_) | MessageContent(_) | MessageEdited(_) | MessageSendSucceeded(_) => {
+                self.history().handle_update(update);
+            }
+            NewMessage(ref data) => {
+                let message = Message::new(data.message.clone(), self);
```

Comment:
```suggestion
            NewMessage(data) => {
                let message = Message::new(data.message, self);
```

Reply:
I can't handle update in that case
![image](https://user-images.githubusercontent.com/63008755/218306256-d3a0e3b0-3116-45f7-a586-5960cf046317.png)